scalecube-organization/pom.xml

```diff
@@ -11,11 +13,10 @@
   <packaging>jar</packaging>
 
   <name>scalecube-organization</name>
-  <url>http://maven.apache.org</url>
 
   <properties>
-    <serviceName>${project.artifactId}</serviceName>
     <mainClass>io.scalecube.server.OrganizationServiceRunner</mainClass>
+    <extratags>${env.TRAVIS_COMMIT}</extratags>
```

Comment:
did you test it? does it deploy each commit to docker hub? is it what you want?

Reply:
> is it what you want?

It is common practice for scalecube services, isn't it?

Reply:
if it is required from a service to be tested as a container across multiple versions (mostly BM testing) we are using it. ask @artem-v